### ISSUE TITLE
No invader tokens from fort (for the 5th time)

### ISSUE THREAD

**Issue by todngo** [WarEmu/WarBugs#15207]
No tokens from stonewatch last night, and this is the fifth time to contact you guys on this, and still waiting for tokens from two of them.  This is getting ridiculous, invader tokens are literally the sole means of progression, so hours and horus of play gets me literally no rewards other than renoun.  I love the game but this is getting retarded.  I'd rather pay a subscription monthly than deal with this.  You owe me a minimum of 12 tokens.  If I cannot progress after spending 5 hours getting to the position of taking a fort, being successfull in it, then getting nothing, why play?

**Comment by raptrr** [comment]
This is providing us no information at all, so we cannot see what happened in your case.
We know there are issues, but we need as much information as possible to resolve this.

**Comment by Natherul** [comment]
Please try and keep feelings away from the bugtracker. All we want is hard info to fix issues.

So closing this ticket for now, feel free to make a new one if you can provide more information, like did you get the debug message? if so how did you enter fort and so on. 

Also note that the bugtracker is not to refund or give compensation in any way. Its for the purpose to fix problems.

**Comment by todngo** [comment]
I've never filed a bug before, so please let me know what you need to know specifically.  Just because I play computer games doesn't mean I know the development\Dev side.

**Comment by todngo** [comment]
If the in game help doesn't fix one small portion of this issue by rewarding tokens that were repeated & rightfully earned, and it's also not done here, then there's no recourse to players to all?  

Most people when having a conversation ask questions to get the info they need, rather than just state their missing some facts and then say goodbye.  So I wasted my time making an account and telling you of an issue, and you wasted your time stating I didnt give the details you needed.  It's like talking to the MS DOS operating system.

Playing my 70 Magus (Kharmah) I entered the fort the second it was open, waited until the outer doors were down, fought amongst several warbands to eventually beat the lord, waited until all bags were distributed, got no errors, checked my invader token count and it was the exact same as when I started like has happened many times before.  There's no screenshots to give you as there were no error messages.  What info are you missing?
